### PR TITLE
XYZ quantum sim tutorial

### PR DESCRIPTION
**Pull request by lhenriet**
Hello,
1) A new tutorial on Floquet engineering with XY mode.
2) Changing the name of the QEK tutorial for consistency.

### CODE REVIEW

File: tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb [hunk 1]
```diff
@@ -0,0 +1,676 @@
+{
```
Comment (Louis-PaulHenry) [inline tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb:1]: <p>Introduce the DW acronym for unfamiliar readers ?</p><p> </p><p>... domain-wall (DW) ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (Louis-PaulHenry) [inline tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb:1]: <p>Maybe add PBC and OBC as a title on each plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (Louis-PaulHenry) [inline tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb:1]: <p>\lange N_{flip} \rangle ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (Louis-PaulHenry) [inline tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb:1]: <p>Is it worth plotting the same quantity in the obc alongside the pbc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (Louis-PaulHenry) [inline tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.        if m 


 m &lt; 3?
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (Louis-PaulHenry) [inline tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb:1]: <p>Nice set of plots!</p><p>Comment on the large black bar in the final panel? What state is it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

**Comment by Louis-PaulHenry** [comment]
Noting much to change! I only have a few remarks/comments. Otherwise LGTM!

File: tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb (hunk 1 above)
Comment (lhenriet) [reply tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb:1]: <p>|1111000001&gt; ! Still a domain wall state, but with two additional spins up. Don't really know why we observe that. But it is actually also something that was observed experimentally if I recall correctly.</p>

Comment (Louis-PaulHenry) [reply tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb:1]: <p>I would then add a legend</p>

**Comment by Louis-PaulHenry** [comment]
Except for the remark on the legend in the plot of the correlations, everything is fine!

**Comment by lhenriet** [comment]
@Louis-PaulHenry, good to go ?

**Review by Louis-PaulHenry — APPROVED** [review]
LGTM!

**Pull request merged**